src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -62,7 +62,9 @@ export class DictionaryModal extends React.Component {
     if (Object.keys(errors).length === 0) {
       this.props
         .submit(this.state.data)
-        .then(() => this.hideModal())
+        .then(() => {
```

Comment:
Is this also about testing the untested areas?

Reply:
No, this is a formatting due to eslint, let me remove it.